src/network_event/client_event.rs

```diff
@@ -220,6 +221,16 @@ fn local_resending_system<T: Event>(
     }
 }
 
+/// Discards all pending events.
+///
+/// We discard events while waiting to connect to ensure clean reconnects.
+fn reset_system<T: Event>(mut events: ResMut<Events<T>>) {
+    let num_drained = events.drain().count();
+    if num_drained > 0 {
+        warn!("discarded {num_drained} client events due to a disconnect");
+    }
```

Comment:
For consistency, I usually use `count`.
```suggestion
fn reset_system<T: Event>(mut events: ResMut<Events<T>>) {
    let drained_count = events.drain().count();
    if drained_count > 0 {
        warn!("discarded {drained_count} client events due to a disconnect");
    }
```